tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "noImplicitAny": false,
     "outDir": "lib",
     "resolveJsonModule": true,
+    "rootDir": ".",
```

Comment:
To resolve URL for CDN, we have to load `package.json` placed on the project root. Thus, `rootDir` was changed from `src` to `.`. It would change the structure of `types` directory too.